test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -4,13 +4,13 @@ set -e
 install_kfctl(){
     #install kfctl cli
     KFCTL_VERSION="v1.0.2"
-    S3_BUCKET="kubeflow-kfctl-binary"
+    S3_URL="https://kubeflow-kfctl-binary.s3-us-west-2.amazonaws.com/kfctl_v1.0.2-1-g93e95e1_linux.tar.gz"
 
     if ! command -v kfctl &> /dev/null
     then
-        aws s3 cp s3://${S3_BUCKET}/kfctl_${KFCTL_VERSION}-1-g93e95e1_linux.tar.gz /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
+        wget -O /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz ${S3_URL}
         tar -xvf /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz -C /tmp --strip-components=1
-        mv /tmp/kfctl /usr/local/bin
+        sudo mv /tmp/kfctl /usr/local/bin
```

Comment:
This script will be running as a part of CDK (lambda function) and `sudo` is required while running on EC2 instance.
Note: The PR builds will have failures as sudo is not supported.